src/backend/expungeservice/record_summarizer.py

```diff
@@ -86,3 +91,20 @@ def get_label(charge: Charge):
             ]
             eligible_charges_by_date[label] = charges_tuples
         return eligible_charges_by_date
+
+    @staticmethod
+    def _build_eligible_nonconvictions_only(charges_eligible_now):
+        return (
+            charges_eligible_now
+            and len(
+                list(
+                    filter(
```

Comment:
Why doesn't a generator have a builtin `next` alternate that returns None instead of an exception? Blar.

Reply:
```
g = iter([0,1])
next(g, None)
```

Reply:
Oh right, We've done this before. I was looking around SO and saw many answers of "yep can't do that" for some reason.